conllulex/conllulex_to_json.py

```diff
@@ -131,7 +131,7 @@ def _store_morph_and_deps(token_dict, token, errors, is_ellipsis, is_supertoken,
         token_dict["deprel"] = None
 
 
-def _store_conllulex_columns(sentence, token_dict, token, errors, ss_mapper):
+def _store_conllulex_columns(sentence, token_dict, token, errors, ss_mapper,corpus='streusle'):
```

Comment:
why is the default value for `corpus` needed here? This seems like an easy way to get subtle bugs--if corpus name is needed, it should be passed in explicitly always, not defaulted to something else. I see you're always passing in the value of `corpus` elsewhere when this function is invoked, so I'd recommend removing the `='streusle'` here

Reply:
Yeah, it's not needed, i've removed it in the latest push now.